test/pytest/test_sm_mme_requirements.py

```diff
@@ -58,7 +58,7 @@ def test_oom_on_model_load():
         "model_name": "BERTSeqClassification",
         "url": "https://torchserve.pytorch.org/mar_files/BERTSeqClassification.mar",
         "batch_size": 1,
-        "initial_workers": 16,
+        "initial_workers": 32,
```

Comment:
this test has been giving us issues for a long time - @lxning do we need to keep this around? I'd rather we skip this test until we find a way to make it more reliable otherwise it makes it unsafe to merge anything since regression test has been red for as long as I can remember at this point https://github.com/pytorch/serve/actions/workflows/regression_tests_gpu_binaries.yml

Reply:
skipping it for now